pubspec.yaml

```diff
@@ -3,10 +3,10 @@ description: Port of https://github.com/PolymerLabs/polymer-snippets
 environment:
   sdk: '>=1.8.2 <2.0.0'
 dependencies:
-  polymer: ^1.0.0-rc.6
-  polymer_elements: ^1.0.0-rc.3
-  reflectable: ^0.3.3
-  web_components: ^0.12.0
+  polymer: ^1.0.0-rc.12
+  polymer_elements: ^1.0.0-rc.5
+  reflectable: ^0.5.1
+  web_components: ^0.12.2
 transformers:
 - web_components:
```

Comment:
Lets also update to the new polymer transformer (available as of 1.0.0-rc.11). Basically just delete the reflectable transformer section (and the reflectable dependency), and then replace the web_components transformer with the polymer one (options are the same)